include/warehouse_ros/impl/message_collection_impl.hpp

```diff
@@ -74,16 +78,27 @@ void MessageCollection<M>::insert(const M& msg, Metadata::Ptr metadata)
   if (!md5sum_matches_)
     throw Md5SumException("Cannot insert additional elements.");
 
-  metadata->append("creation_time", ros::Time::now().toSec());
+  metadata->append("creation_time", rclcpp::Clock().now().seconds());
```

Comment:
Seems `RCL_ROS_TIME` is the default argument when `Clock` is initialized. But I think it's better to declare it explicitly. Change done.

Reply:
I think nodes use RCL_ROS_TIME by default, Clock uses system time (https://github.com/ros2/rclcpp/blob/master/rclcpp/include/rclcpp/clock.hpp#L66). I'm still not totally sure when we should use which since many ROS2 packages use default clock constructor now.

Reply:
I was wrong, the clock does use `RCL_SYSTEM_TIME` by default. From the answers to this question (https://answers.ros.org/question/287946/ros-2-time-handling/), seems only the clock initialized from a node can use the simulated time source. Thus, I think rclcpp::clock (`RCL_ROS_TIME`) eventually equals to a clock with type `RCL_SYSTEM_TIME`. I'm not sure about it. I will dig some further to check it.